src/CultureFeed/CultureFeedConsumerAdapter.php

```diff
@@ -56,4 +56,14 @@ public function getName(): ?string
     {
         return  $this->cfConsumer->name;
     }
+
+    public function isBlocked(): bool
+    {
+        return isset($this->cfConsumer->status) && $this->cfConsumer->status === 'BLOCKED';
```

Comment:
Would it be safer to assume the consumer was blocked when it has no status?

Reply:
To be honest I'm not sure, it's a pretty old XML API and I'm just using a very old PHP library to communicate with it here...

Technically it's the same as before though: https://github.com/cultuurnet/udb3-api-guard/blob/main/src/CultureFeed/CultureFeedApiKeyAuthenticator.php#L67-L73

Before these changes, it just checked if the `$status` was either `BLOCKED` or `REMOVED` and if so threw an exception. But if the `$status` was `null` it would not throw an exception.

So to err on the side of caution it's probably best to not throw if there's no status?

I can remove the `isset($this->cfConsumer->status)` if you want to make it exactly the same comparison as before though.

Reply:
I guess matching the old implementation as close as possible is best?

Reply:
It already matched the old implementation functionally but I've removed the unnecessary `isset()`s to be sure!